src/main/resources/com/openshift/jenkins/plugins/OpenShiftDSL.groovy

```diff
@@ -1141,13 +1142,19 @@ class OpenShiftDSL implements Serializable {
 
         public OpenShiftResourceSelector startBuild(Object... ouserArgs ) throws AbortException {
             String[] userArgs = toStringArray(ouserArgs);
+            List argList = Arrays.asList(userArgs);
+            boolean realTimeLogs = argList.contains("-F") || argList.contains("--follow=true") || argList.contains("--follow");
```

Comment:
if realTimeLogs, I would not call splitNames on the output. This will create a selector that tries to select each line of log output as if it were a named openshift resource. This is the primary reason folks shouldn't specify -F on startBuild (and should use .logs()) --  the selector it returns will not be of any use.

Reply:
thanks for taking a look @jupierce 

First, to confirm, since we last talked and agreed to adding the -F / --follow support, you are still OK with the fact we added it, but you are noting some of the side effects, and clarifying that .logs() is preferred (which was my take from our original conversation).  Yes?

I'll open a separate pull shortly to address the `splitNames` piece.